scripts/apps/search/directives/SearchParameters.js

```diff
@@ -28,7 +28,12 @@ export function SearchParameters($location, asset, tags, metadata, meta, common,
                 scope.query = params.q;
                 scope.flags = false;
                 scope.common = common;
-                scope.meta = meta = {};
+                scope.meta = meta;
+                for (var member in meta) {
```

Comment:
on the last P.R. (merged this morning), I did try to reset meta to see if it was fixing  the tests, it did indeed, but because I replaced the meta object, the values were not changed anymore (no more double binding), and it was merged like this.

This P.R. fix it by reseting meta instead of replacing the object, and the default values works again.

Actually all of this is because of some removed code a couple of weeks ago, resulting in default values not set anymore and https://dev.sourcefabric.org/browse/SDNTB-288 to fix .

Because code was then splitted in directives in a refactoring, I had to find a way to communicate between itemRepo and SearchParameter, which I did using angular.value, but that broke the tests and since then I want to curse everybody involved in javascript creation and ecosystem.